main.go

```diff
@@ -217,6 +218,20 @@ func main() {
 
 	otel.SetErrorHandler(otelErrorHandler{logger: logger})
 
+	reg := prometheus.NewRegistry()
+	reg.MustRegister(
+		version.NewCollector("observatorium"),
+		collectors.NewGoCollector(),
+		collectors.NewProcessCollector(collectors.ProcessCollectorOpts{}),
+	)
+
+	skippedTenants := promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+		Namespace: "observatorium",
+		Subsystem: "api",
+		Name:      "tenants_skipped_invalid_configuration",
```

Comment:
My only thought here is whether `tenants_skipped` is slightly ambiguous - perhaps `tenants_invalid_configuration` is more direct? If a tenant has invalid configuration - they are not being registered!

Reply:
Not a blocker tho :man_shrugging: